docs/howtos/index.md

```diff
@@ -0,0 +1,30 @@
+# How-to Guides
+
+!!! warning "🚧 Work in Progress"
```

Comment:
I would not add warnings like this one, since the warning is not likely to ever get removed as the documentation is likely to always be WIP (there's always an extra chapter that can be written). Or rather, if we add a warning, there should be a clear definition of done and hence when the warning can be removed.